master/buildbot/test/unit/test_worker_docker.py

```diff
@@ -61,7 +61,7 @@ def test_constructor_minimal(self):
 
     def test_constructor_all_docker_parameters(self):
         # Volumes have their own tests
-        bs = self.ConcreteWorker('bot', 'pass', 'unix:///var/run/docker.sock', 'worker_img', ['/bin/sh'], dockerfile="FROM ubuntu", version='1.9', tls=True)
+        bs = self.ConcreteWorker('bot', 'pass', 'unix:///var/run/docker.sock', 'worker_img', ['/bin/sh'], dockerfile="FROM ubuntu", version='1.9', tls=True, hostconfig={'network_mode': 'fake', 'dns': ['1.1.1.1', '1.2.3.4.']})
```

Comment:
Please arrange arguments over several lines, I think in general maximum 80 or 100 characters per line convention is used.

Do you intentionally used invalid IP address string here: `1.2.3.4.` (with dot at the end)?